ci/launch_cluster.sh

```diff
@@ -26,49 +26,36 @@ fi
 CHANNEL="$1"
 VARIANT="$2"
 DEPLOYMENT_NAME="$3"
-TERRAFORM_VARS="$DEPLOYMENT_NAME.tfvars"
 TERRAFORM_STATE="$4"
 
 if [ "$VARIANT" == "open" ]; then
   INSTALLER="https://downloads.dcos.io/dcos/${CHANNEL}/dcos_generate_config.sh"
-  TERRAFORM_MODULE="github.com/dcos/terraform-dcos/aws"
 else
   INSTALLER="https://downloads.mesosphere.com/dcos-enterprise/${CHANNEL}/dcos_generate_config.ee.sh"
-  TERRAFORM_MODULE="git@github.com:mesosphere/enterprise-terraform-dcos/aws"
 fi
 
 echo "Using: ${INSTALLER}"
 
-rm -rf "terraform-plan"
-mkdir "terraform-plan"
-terraform init -from-module "$TERRAFORM_MODULE" "terraform-plan"
-
+# Create cluster.
+terraform init
+terraform apply -auto-approve -state "$TERRAFORM_STATE" \
+	-var "cluster_name=\"$DEPLOYMENT_NAME\"" \
+	-var "dcos_variant=\"$VARIANT\""
 # Create .tfvars file for terraform.
-envsubst <<EOF > "$TERRAFORM_VARS"
-dcos_cluster_name = "$DEPLOYMENT_NAME"
-custom_dcos_download_path = "$INSTALLER"
-num_of_masters = "3"
-num_of_private_agents = "3"
-num_of_public_agents = "1"
-aws_region = "us-west-2"
-aws_bootstrap_instance_type = "m4.large"
-aws_master_instance_type = "m4.large"
-aws_agent_instance_type = "m4.large"
-aws_public_agent_instance_type = "m4.large"
-os = "centos_7.2"
-ssh_key_name = "default"
+#envsubst <<EOF > "$TERRAFORM_VARS"
+#custom_dcos_download_path = "$INSTALLER"
+#ssh_key_name = "default"
 # Inbound Master Access
-admin_cidr = "0.0.0.0/0"
-EOF
+#admin_cidr = "0.0.0.0/0"
+#EOF
 
 # Append license and security mode for EE variants.
-if [ "$VARIANT" != "open" ]; then
-	dcos_security = "$VARIANT"
-	dcos_license_key_contents = "$DCOS_LICENSE"
-fi
+# if [ "$VARIANT" != "open" ]; then
+#	dcos_security = "$VARIANT"
+#	dcos_license_key_contents = "$DCOS_LICENSE"
+#fi
 
-# Create cluster.
```

Comment:
Where is cluster deletion with terraform?

Reply:
In `si_pipeline.sh`. It's still a little weird that we have to pass variables.